packages/upgrade/package.json

```diff
@@ -30,22 +31,20 @@
     "access": "public"
   },
   "scripts": {
-    "build": "esbuild src/cli.js --bundle --platform=node --outfile=cli.js --target=node14",
+    "build": "esbuild src/cli.js --bundle --platform=node --outfile=cli.js --target=node14 --external:jscodeshift",
     "clean": "rimraf cli.js",
     "watch": "yarn build --watch"
   },
   "devDependencies": {
     "chalk": "^4.1.1",
     "change-case": "^4.1.2",
-    "cross-spawn": "^7.0.3",
     "esbuild": "^0.14.10",
     "execa": "^5.1.1",
```

Comment:
@tay1orjones running into this issue when running a specific codemod:

![image](https://user-images.githubusercontent.com/3901764/151226887-1cdd43ed-6d10-48e0-825b-a44cbe55356f.png)

I think this is because `execa` is specified as `^5.1.1` where it is treated as a default export versus a named export in v6+

Reply:
Agh yeah - sorry this was from when I was looking at upping the dep to v6. Decided to keep it at v5 and forgot to change the usage. I'll push an update.

Reply:
@joshblack updated 👍